react/react-training/src/redux/game/actions.js

```diff
@@ -0,0 +1,26 @@
+// noinspection JSUnusedGlobalSymbols
+export const makeMove = (history, squares, stepNumber, xIsNext) => {
```

Comment:
Move this function and `makeJump` inside of a object called `actionsCreators` and export it